config/security.js

```diff
@@ -1,69 +1,47 @@
-    // credentials: true,
-    /***************************************************************************
-     *                                                                          *
-     * Which methods should be allowed for CORS requests? This is only used in  *
-     * response to preflight requests (see article linked above for more info)  *
-     *                                                                          *
-     ***************************************************************************/
-    // methods: 'GET, POST, PUT, DELETE, OPTIONS, HEAD',
-    /***************************************************************************
-     *                                                                          *
-     * Which headers should be allowed for CORS requests? This is only used in  *
-     * response to preflight requests.                                          *
-     *                                                                          *
-     ***************************************************************************/
-    // headers: 'content-type'
-  },
+  /***************************************************************************
+   *                                                                          *
+   * CORS is like a more modern version of JSONP-- it allows your application *
+   * to circumvent browsers' same-origin policy, so that the responses from   *
+   * your Sails app hosted on one domain (e.g. example.com) can be received   *
+   * in the client-side JavaScript code from a page you trust hosted on _some *
+   * other_ domain (e.g. trustedsite.net).                                    *
+   *                                                                          *
+   * For additional options and more information, see:                        *
+   * https://sailsjs.com/docs/concepts/security/cors                          *
+   *                                                                          *
+   ***************************************************************************/
+
+  // cors: {
+  //   allRoutes: false,
+  //   allowOrigins: '*',
+  //   allowCredentials: false,
+  // },
+
+  /****************************************************************************
+   *                                                                           *
+   * CSRF protection should be enabled for this application.                   *
+   *                                                                           *
+   * For more information, see:                                                *
+   * https://sailsjs.com/docs/concepts/security/csrf                           *
+   *                                                                           *
+   ****************************************************************************/
+
+  // TODO: We should look in to enabling this for security reasons in the future
+  csrf: false,
```

Comment:
The example app had this as true but I don't think we can do this yet due to the way SSL is handled?